packages/components/src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import isChromatic from "chromatic/isChromatic";
+import { Button } from "../Button";
+import { Toast } from "./Toast";
+import { ToastViewport } from "./ToastViewport";
+import { ToastProvider } from ".";
+
+export default {
+  component: Toast,
+  title: "Components/Toast",
+} as ComponentMeta<typeof Toast>;
+
+const Template: ComponentStory<typeof Toast> = (args) => {
+  const [open, setOpen] = React.useState(isChromatic() ? true : false);
+
+  return (
+    <ToastProvider duration={5000}>
```

Comment:
In actual usage, would we need to wrap each instance with the ToastProvider and ToastViewport or would there just be one that wraps the whole app? Might be good to add that info the documentation.

Reply:
`ToastProvider` and ToastViewport` will only need to be added once to the app. I'll add a comment in the stories.